frontend/views/containers/chatroom/ViewArea.vue

```diff
@@ -0,0 +1,93 @@
+<template lang='pug'>
+.c-view-wrapper
+  p.c-view-label
+    span You are viewing{{' '}}
+    b {{'#' + title}}
```

Comment:
Strings that have formatting should be constructed as a complete full sentence using `i18n` or `L` function so that they can be translated, using the appropriate formatting inside of the sentence, e.g. `{b}bold text{_b}` etc.

Reply:
Thanks for your catch. I missed it to translate.